app/assets/stylesheets/modules/master_overlay.css.scss

```diff
@@ -394,6 +394,8 @@ $parent-width: $parent-split * 0.21;
 
 .details-content {
   white-space: pre;
+  font-family: monospace;
```

Comment:
This also affects CMR granule details but I think make it monospace the indentation will be more clear.